library/ssl_tls12_server.c

```diff
@@ -3404,9 +3404,15 @@ static int ssl_decrypt_encrypted_pms( mbedtls_ssl_context *ssl,
                                       size_t *peer_pmslen,
                                       size_t peer_pmssize )
 {
-    int ret = MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED;
```

Comment:
We normally always initialise return values to MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED in this manner - so I don't see any reason to change this.

Reply:
This change is in response to a separate Coverity warning "value not used", which rightfully indicates that the variable will be assigned a different value before it is read from the first time. Normally I would just ignore such a low level warning, but since I was touching this function anyway, I decide to address this one too.

Reply:
Initializing variables to a constant regardless of how they're used later is very common practice. Some coding rules even forbid uninitialized variables (ours doesn't: we think that the best practice depends on how the variable is used.)

This particular idiom with `ret` and `status` variables is used throughout Mbed TLS. Is your Coverity instance complaining about all of them? (Ours isn't.)